### DISCUSSION TITLE
default password?

### DISCUSSION THREAD

**Discussion by unecc** [community-scripts/ProxmoxVE#857] [General]
Hi what's default password of each vm?

**Comment by stocky789** [comment]
i want to know the same thing 
i would of thought the script would spit this out at the end with the URL 

not much use if you cant login to it xD

**Reply by michelroegl-brunner** [reply]
When you install a new LXC/VM with default settings and there are no credentials on the homepage, you should get logged in to root once you enter the shell via proxmox web ui.

**Reply by unecc** [reply]
Not true many default settings, set password ;)  try yuno after installed ask login

**Comment by MickLesk** [comment]
Which VMS? 

All Logins are called in website. If you dont set an password, its root and empty pw.

**Reply by unecc** [reply]
it's not true

![image](https://github.com/user-attachments/assets/0eef2d9f-eb66-4e9d-99fc-01bd57037b6d)

**Reply by michelroegl-brunner** [reply]
Wich script are you using and are you using default or advanced settings. We can not help you when we have no information.

**Reply by unecc** [reply]
it's written in screenshot. yunohost.

**Reply by michelroegl-brunner** [reply]
default or advanced? Else i can't test it/help you.

**Reply by unecc** [reply]
all default setting

**Reply by MickLesk** [reply]
Yunohost is not an VM.

**Reply by MickLesk** [reply]
Its not an new Installation? Its Debian 12 Not 11.

Check yunohost Docs for Support 

https://yunohost.org/#/ssh
